lib/src/instance/get_instance.dart

```diff
@@ -69,21 +82,26 @@ class GetInstance {
         isSingleton: false, name: name, builder: builder, permanent: permanent);
   }
 
+  /// Injects the Instance [S] builder into the [_singleton] HashMap.
+  ///
   void _insert<S>({
     bool isSingleton,
     String name,
     bool permanent = false,
     FcBuilderFunc<S> builder,
   }) {
     assert(builder != null);
-    String key = _getKey(S, name);
-
+    final key = _getKey(S, name);
     _singl.putIfAbsent(
         key, () => FcBuilder<S>(isSingleton, builder, permanent, false));
   }
 
+  /// Clears from memory registered Instances associated with [routeName] when
+  /// using [GetConfig.smartManagement] as [SmartManagement.full] or [SmartManagement.keepFactory]
+  /// Meant for internal usage of [GetPageRoute] and [GetDialogRoute]
+  ///
```

Comment:
This is just a aesthetical suggestion, but i think is not nice to leave a blank line between the docs and the methods that is being documented.
Since the doc are directly connected to the method, they should be close. 
But that's just my opinion, if you prefer the way it is (and id fine for @jonataslaw ), it can remain as is.

Reply:
No problem, I didn't see any standards for it, code was more readable for me. 
It would cool to have some feedback about it though, if it can work as a foundation for a proper docs later at least.